### PR TITLE
EZP-24836: Add the navigation zone item to the Users subtree

### PR DESCRIPTION
**Pull request by yannickroger**
Link: https://jira.ez.no/browse/EZP-24836

## Description
Created a static entry for the "Users" menu.

## Screenshot
users_hub

## Tests
Manual and updated unit tests

### CODE REVIEW

**Comment by yannickroger** [comment]
ping @dpobel @mhyndle @StephaneDiot

**Comment by StephaneDiot** [comment]
+1

**Comment by mhyndle** [comment]
+1

**Comment by yannickroger** [comment]
I'll still wait for the +1 of our guru @dpobel  :)

**Comment by andrerom** [comment]
+1

File: Tests/js/views/services/assets/ez-navigationhubviewservice-tests.js [hunk 1]
```diff
@@ -562,8 +562,16 @@ YUI.add('ez-navigationhubviewservice-tests', function (Y) {
             this.service.removeNavigationItem(identifier, zone);
 
             Y.Array.each(this.service.get(zone + "NavigationItems"), function (item) {
+                var itemIdentifier;
+
+                if (item.config) {
+                    itemIdentifier = item.config.identifier;
+                } else {
+                    itemIdentifier = item.identifier; //subtreeItem
```
Comment (dpobel) [inline Tests/js/views/services/assets/ez-navigationhubviewservice-tests.js]: weird patch, I think in this case item is an instance of eZ.NavigationItemSubtreeView so to get it's identifier you should write `item.get('identifier')`

File: Tests/js/views/services/assets/ez-navigationhubviewservice-tests.js [hunk 2]
```diff
@@ -562,8 +562,16 @@ YUI.add('ez-navigationhubviewservice-tests', function (Y) {
             this.service.removeNavigationItem(identifier, zone);
 
             Y.Array.each(this.service.get(zone + "NavigationItems"), function (item) {
+                var itemIdentifier;
+
+                if (item.config) {
```
Comment (dpobel) [inline Tests/js/views/services/assets/ez-navigationhubviewservice-tests.js]: I would rather test if `item` is an instance of `Y.eZ.NavigationItemSubtreeView`

**Comment by yannickroger** [comment]
Weirdness fixed @dpobel

**Comment by dpobel** [comment]
+1

**Pull request merged**